tests/console/wavpack.pm

```diff
@@ -34,17 +34,12 @@ sub run {
     select_console 'root-console';
     # development module needed for dependencies, released products are tested with sdk module
     if (is_sle) {
-        if (get_var('BETA')) {
-            my $sdk_repo = is_sle('15+') ? get_var('REPO_SLE_MODULE_DEVELOPMENT_TOOLS') : get_var('REPO_SLE_SDK');
-            zypper_ar 'http://' . get_var('OPENQA_URL') . "/assets/repo/$sdk_repo", name => "SDK";
-        }
-        # maintenance updates are registered with sdk module
-        elsif (get_var('FLAVOR') !~ /Updates|Incidents/) {
+        if (get_var('FLAVOR') !~ /Updates|Incidents/) {
             cleanup_registration;
             register_product;
-            add_suseconnect_product('sle-module-desktop-applications');
-            add_suseconnect_product(get_addon_fullname('sdk'));
         }
+        add_suseconnect_product('sle-module-desktop-applications');
```

Comment:
QAM tests does not need to add sdk, they already have it.

Reply:
leave it where it was `if (get_var('FLAVOR') !~ /Updates|Incidents/) {`

Reply:
sure I was just trying to keep things for QAM untouched , this is how it was before . Do you want me to fully drop this for QAM ? Can you point me which job I should clone to check if it is working as expected ?

Reply:
Did you mean desktop module instead of SDK?
Because, before this PR the test was adding only SDK for QAM tests.

Reply:
no, it was adding sdk for QA, not for QAM tests

Reply:
Right, QAM is not BETA=1, I was confused.

Both lines should be inside the if.

```perl
if (get_var('FLAVOR') !~ /Updates|Incidents/) {
  cleanup_registration;
  register_product;
  add_suseconnect_product('sle-module-desktop-applications');
  add_suseconnect_product(get_addon_fullname('sdk'));
}
```

Reply:
@asmorodskyi https://openqa.suse.de/tests/3709348, but when its working on QA tests then it's ok,
again, all this code is there only because of QA tests. QAM, does have sdk there by default.

Reply:
how about this version ?

Reply:
This is good